model.py

```diff
@@ -3358,7 +3358,7 @@ def choose_cover(self):
                     if not rep.mirrored_at and not rep.thumbnails:
                         logging.warn(
                             "Best cover for %r (%s) was never mirrored or thumbnailed!",
-                            self.primary_identiifer,
+                            self.primary_identifer,
```

Comment:
This is still wrong; should be `primary_identifier`.

Reply:
Opsss ! Changed it !